module/batch-action/src/Infrastructure/JMS/Serializer/Handler/BatchActionFilterIdsFormModelHandler.php

```diff
@@ -74,9 +75,17 @@ public function deserialize(
         if (is_array($data['list'] ?? null)) {
             $model->list = $data['list'];
         }
-        if (is_bool($data['included'] ?? null)) {
-            $model->included = $data['included'];
+        switch ($data['included']) {
```

Comment:
The current behavior should be left as it was - we won't be getting strings in place of booleans in POST's payload